src/pages/ReimbursementAccount/BankAccountPlaidStep.js

```diff
@@ -39,104 +39,99 @@ const defaultProps = {
-                        text={this.props.translate('bankAccount.plaidBodyCopy')}
-                        onSelect={(plaidAccountID) => {
-                            ReimbursementAccount.updateReimbursementAccountDraft({plaidAccountID});
-                        }}
-                        plaidData={this.props.plaidData}
-                        onExitPlaid={() => BankAccounts.setBankAccountSubStep(null)}
-                        receivedRedirectURI={this.props.receivedRedirectURI}
-                        plaidLinkOAuthToken={this.props.plaidLinkOAuthToken}
-                        allowDebit
-                        bankAccountID={bankAccountID}
-                        selectedPlaidAccountID={selectedPlaidAccountID}
+                {Boolean(selectedPlaidAccountID) && !_.isEmpty(lodashGet(plaidData, 'bankAccounts')) && (
+                    <CheckboxWithLabel
+                        accessibilityLabel={`${props.translate('common.iAcceptThe')} ${props.translate('common.expensifyTermsOfService')}`}
+                        style={styles.mt4}
+                        inputID="acceptTerms"
+                        LabelComponent={() => (
+                            <Text>
+                                {props.translate('common.iAcceptThe')}
+                                <TextLink href={CONST.TERMS_URL}>{props.translate('common.expensifyTermsOfService')}</TextLink>
+                            </Text>
+                        )}
+                        defaultValue={props.getDefaultStateForField('acceptTerms', false)}
+                        shouldSaveDraft
                     />
-                    {Boolean(selectedPlaidAccountID) && !_.isEmpty(lodashGet(this.props.plaidData, 'bankAccounts')) && (
-                        <CheckboxWithLabel
-                            accessibilityLabel={`${this.props.translate('common.iAcceptThe')} ${this.props.translate('common.expensifyTermsOfService')}`}
-                            style={styles.mt4}
-                            inputID="acceptTerms"
-                            LabelComponent={() => (
-                                <Text>
-                                    {this.props.translate('common.iAcceptThe')}
-                                    <TextLink href={CONST.TERMS_URL}>{this.props.translate('common.expensifyTermsOfService')}</TextLink>
-                                </Text>
-                            )}
-                            defaultValue={this.props.getDefaultStateForField('acceptTerms', false)}
-                            shouldSaveDraft
-                        />
-                    )}
-                </Form>
-            </ScreenWrapper>
-        );
-    }
+                )}
+            </Form>
+        </ScreenWrapper>
+    );
 }
 
 BankAccountPlaidStep.propTypes = propTypes;
 BankAccountPlaidStep.defaultProps = defaultProps;
+BankAccountPlaidStep.displayName = 'BankAccountPlaidStep';
 export default compose(
     withLocalize,
```

Comment:
We can also use the [`useLocalize`](https://github.com/Expensify/App/blob/206f107a09ee82a5ede8383ee7f293738618a7b1/src/hooks/useLocalize.js#L4) custom hook instead now.

Reply:
Make sense bump @payal-lathidadiya

Reply:
updated

Reply:
@Santhosh-Sellavel do you want me to raise a quick PR to update that?

Reply:
Yes

Reply:
raised fix here - https://github.com/Expensify/App/pull/22473